cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -442,6 +452,21 @@ class CreateTaskContent extends React.PureComponent<Props & RouteComponentProps,
             return;
         }
 
+        try {
+            this.validateQualityParams(quality);
+        } catch (e) {
+            if (e instanceof Error) {
+                notification.error({
+                    message: 'Could not create a task',
+                    description: e.message,
+                    className: 'cvat-notification-create-task-fail',
+                });
+                reject();
+                return;
+            }
```

Comment:
Do not we reject/return if because of some reason e is not instance of Error? 
```suggestion
            notification.error({
                message: 'Could not create a task',
                description: e instanceof Error ? e.message : '',
                className: 'cvat-notification-create-task-fail',
            });
            reject();
            return;
        }
```